examples/configuration.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: pkg.crossplane.io/v1
 kind: Configuration
 metadata:
-  name: cofiguration-aws-eks
+  name: configuration-aws-eks
 spec:
   package: xpkg.upbound.io/upbound/configuration-aws-eks:v0.12.0
```

Comment:
can we bump the version here while we have this unique opportunity? :D

Reply:
yes its the new version